examples/local/scripts/ovttablet-up.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
```

Comment:
Can you please explain what `ovttablet-up.sh` is for? How does it differ from `vttablet-up.sh`?

Reply:
I will explain here (because it's a temporary file). The two changes are:
* Removed `enable_semi_sync`. This tells vttablet not to touch semi-sync settings.
* Added `disable_active_reparents=true`. This tells vttablet not to touch replication.

Reply:
:+1: